packages/coinstac-client-core/src/index.js

```diff
@@ -214,11 +216,7 @@ class CoinstacClient {
 
       for (let i = 0; i < filesArray.length; i += 1) {
         linkPromises.push(
-          fs.link(
-            filesArray[i],
-            `${this.appDirectory}/${this.clientId}/${runId}/${filesArray[i].replace(/\//g, '-')}`,
-            err => console.log(err) // eslint-disable-line no-console
-          )
+          linkAsync(filesArray[i], `${this.appDirectory}/${this.clientId}/${runId}/${filesArray[i].replace(/\//g, '-')}`)
```

Comment:
Check any other instances of this if you can

Reply:
The same issue is likely found in the `unlinkFiles` function. I'll take a look.